### ISSUE TITLE
Livelossplot logger

### ISSUE THREAD

**Issue by Bartolo1024** [pytorch/ignite#1069]
## 🚀 Feature - Livelossplot logger

The logger that is similar to Neptune, W&B, tensorboard loggers available in Ignite.
livelossplot:
[https://github.com/stared/livelossplot](url)
ignite plugin that is available in the livelossplot:[https://github.com/stared/livelossplot/blob/master/livelossplot/inputs/pytorch_ignite.py](url)
notebook example:
[https://github.com/stared/livelossplot/blob/master/examples/pytorch-ignite.ipynb](url)
The logger can be extended to log other things in addition to metrics (for ex. learning rate).

**Comment by sdesrozis** [comment]
@Bartolo1024 Thank you for this feature !! You already discussed with @vfdev-5 about this, we need to open a PR to discuss deeper :)

Do you want to contribute on it ? It would be a pleasure for us :)

**Comment by vfdev-5** [comment]
@Bartolo1024 thanks for opening FR here !

Let's discuss here a detail about your comment: https://github.com/stared/livelossplot/pull/105#issuecomment-633967435

Is it possible to do If we would like to log `lr` or batch loss per iteration in addition to logging metrics per epoch ? 

And yes, as @sdesrozis proposed, feel free to send a PR if you would like to contribute !

**Comment by Bartolo1024** [comment]
> Is it possible to do If we would like to log `lr` or batch loss per iteration in addition to logging metrics per epoch?
Yes, it is possible - each LogItem has value and step, which can be specified in `plotlosses.update` method.
> And yes, as @sdesrozis proposed, feel free to send a PR if you would like to contribute !
Of course, it will be an awesome experience. I will prepare a PR :)

**Comment by vfdev-5** [comment]
Hey @Bartolo1024 how about the PR :)

**Comment by Bartolo1024** [comment]
Firstly I was busy and after that, I have forgotten about it :( Here is the PR #1167 but I have to add tests. Additionally, I have one question there.
